### DISCUSSION TITLE
Key behaviour after finishing a map

### DISCUSSION THREAD

**Discussion by Bubbleman532** [ppy/osu#25124] [Ideas] [answered]
From stable, the expected behaviour of the esc key after finishing a map is as follows:

- Between the last hitobject and when the scoreboard starts transitioning in: Moves to the start of the scoreboard transitioning in
- Any time after the scoreboard starts showing: Back to song select

However, pressing esc at any point after the end of the final object moves back to song select.

Similarly to this, clicking at any point in the scoreboard animation skips to the end. The bound skip button also skips both the wait between the last object and the scoreboard appearing and the scoreboard animation.

Lazer behaviour in these aspects should match stable.

**Comment by Theighlin** [comment, marked as answer]
Fixed since yesterday (or the prominent part of this) https://github.com/ppy/osu/pull/25097

**Reply by Bubbleman532** [reply]
how did i miss this,,,

**Reply by nagi-desuuu** [reply]
Because it wasn't part of release branch yet...
